src/test/videopress/local-helpers/utils.js

```diff
@@ -48,3 +49,39 @@ export const pressSettingInPanel = async ( screen, panel, setting ) => {
 	// Toggle the specified setting
 	fireEvent.press( getByText( setting ) );
 };
+
+/**
+ * Presses a setting in a specified panel, triggers a picker, and selects an option from the sheet.
+ *
+ * @param {Object}   screen  - The editor's screen.
+ * @param {string}   panel   - The panel containing the setting to be pressed.
+ * @param {string}   setting - The setting to be pressed, which triggers the picker.
+ * @param {string[]} options - An array of all the options available in the picker.
+ * @param {string}   option  - The option to be selected from the picker.
+ */
+export const pressSettingInPicker = async (
+	screen,
+	panel,
+	setting,
+	options,
+	option
+) => {
+	const { getByText } = screen;
+
+	// Navigate to the specified settings panel
+	fireEvent.press( getByText( panel ) );
+
+	// Select the specified setting to trigger picker
+	const pickerButtonPressed = async () => {
+		fireEvent.press( getByText( setting ) );
+	};
+
+	// Setup the picker and select option
+	const { selectOption } = setupPicker(
+		screen,
+		options,
+		pickerButtonPressed
+	);
+
+	await selectOption( option );
```

Comment:
In relation to WordPress/gutenberg#50493, I've made the following change and I didn't experience any issue on iOS with the picker. @SiobhyB not sure if I'm overlooking anything, so I'm wondering if you could share the example that drove you to create WordPress/gutenberg#50493 to have a better context 🤔.

```suggestion
	// Setup the picker and select option
	const { selectOption } = setupPicker( screen, options );

	fireEvent.press( getByText( setting ) );

        // Use `act` here as render updates might happen after selecting an option.
	await act( () => selectOption( option ) );
```

Reply:
Oh, thanks for this Carlos, this stems from me misunderstanding how the `setupPicker` should function. I had been calling the `fireEvent.press( getByText( setting ) )` prior to setting it up, with the believe it was supposed to be set up upon being called. Thank you again for correcting me understanding, I've addressed this in https://github.com/wordpress-mobile/gutenberg-mobile/pull/5754/commits/37692881b062e5329c8c84042e78733460bb203a.